docs/tutorial/updates.md

```diff
@@ -0,0 +1,95 @@
+# Updating Applications
+
+There are several ways to update an Electron application. The easiest and 
+officially supported one is taking advantage of the built-in 
+[Squirrel](https://github.com/Squirrel) framework and 
+Electron's [autoUpdater](../api/auto-updater.md) module.
+
+## Deploying an update server
+
+To get started, you first need to deploy a server that the 
+[autoUpdater](../api/auto-updater.md) module will download new updates from.
+
+Depending on your needs, you can choose from one of these:
+
+- [Hazel](https://github.com/zeit/hazel) – Pulls new releases from 
+[GitHub Releases](https://help.github.com/articles/creating-releases/) and can 
+be deployed for free on [Now](https://zeit.co/now).
+- [Nuts](https://github.com/GitbookIO/nuts) – Also uses 
+[GitHub Releases](https://help.github.com/articles/creating-releases/), 
+but caches app updates on disk and supports private repositories.
+- [electron-release-server](https://github.com/ArekSredzki/electron-release-server) – 
+Provides a dashboard for handling releases
+
+## Implementing updates in your app
+
+Once you've deployed your update server, continue with importing the required 
+modules in your code. The following code might vary for different server 
+software, but it works like described when using 
+[Hazel](https://github.com/zeit/hazel).
+
+**Important:** Please ensure that the code below will only be executed in 
+your packaged app, and not in development. You can use 
+[electron-is-dev](https://github.com/sindresorhus/electron-is-dev) to check for 
+the environment.
+
+```js
+const { app, autoUpdater, dialog } = require('electron')
+```
+
+Next, construct the URL of the update server and tell 
+[autoUpdater](../api/auto-updater.md) about it:
+
+```js
+const server = 'https://your-deployment-url.com'
+const feed = `${server}/update/${process.platform}/${app.getVersion()}`
+
+autoUpdater.setFeedURL(feed)
+```
+
+As the final step, check for updates (the example below will check every minute):
+
+```js
+setInterval(() => {
+  autoUpdater.checkForUpdates()
+}, 60000)
+```
+
+Once your application is [packaged](../tutorial/application-distribution.md), 
+it  will receive an update for each new 
+[GitHub Release](https://help.github.com/articles/creating-releases/) that you 
```

Comment:
I don't have a stance on what the "best" option is, but I think it's useful to get a guide to updating apps off the ground that goes beyond our current API docs for autoUpdater, which leave much to be desired.

Eventually I think it would be great for this document to have a section for each of the update server thingies we want to recommend (nuts, etc), and I'm comfortable using Hazel as the prototype for that.